app/migrations/Version20190402144714.php

```diff
@@ -53,12 +67,15 @@ public function up(Schema $schema): void
           assessor_request_vote_place_wishes 
         ADD 
           CONSTRAINT FK_2EFFDE11F3F90B30 FOREIGN KEY (vote_place_id) REFERENCES vote_place (id) ON DELETE CASCADE');
+        $this->addSql('ALTER TABLE vote_place ADD full TINYINT(1) NOT NULL');
```

Comment:
I think you can avoid adding this column here, and do it in the second migration with a new name directly (to avoir ADD + RENAME queries)

Reply:
We will probably create only one migration when migrating `assessor` branch to `master`